app.py

```diff
@@ -1,20 +1,24 @@
 from flask import Flask
 from dotenv import load_dotenv
 
-from src.controller.review import review
-from src.controller.static_files import static_files
-import src.db.models.UserModel as UserModel
-import src.db.models.ApplicationsModel as ApplicationsModel
+from src.shared import Shared
 
 load_dotenv()
 
-UserModel.initialize_user_model()
-ApplicationsModel.initialize_applications_model()
-
 app = Flask(__name__)
 
 # load config
 app.config.from_object('src.config.config.Config')
 app.secret_key = app.config.get('SECRET_KEY')
+
+# instantiate shared data
+Shared.instantiate(app)
+
+from src.controller.admin import admin
```

Comment:
why are these here?

Reply:
The controllers depend on the shared data instances being instantiated, so I'm importing them after.

Reply:
seems like a circular reference but okie